packages/wonder-blocks-clickable/src/components/__tests__/clickable.test.js

```diff
@@ -452,6 +454,56 @@ describe("Clickable", () => {
         );
     });
 
+    test("should add aria-disabled if disabled is set", () => {
+        // Arrange
+
+        // Act
+        render(
+            <Clickable testId="button" disabled={true}>
+                {(eventState) => <h1>Click Me!</h1>}
+            </Clickable>,
+        );
+
+        const button = screen.getByTestId("button");
+
+        // Assert
+        expect(button).toHaveAttribute("aria-disabled", "true");
+    });
+
+    test("allow keyboard navigation when disabled is set", () => {
+        // Arrange
+        render(
+            <Clickable testId="button" disabled={true}>
+                {(eventState) => <h1>Click Me!</h1>}
+            </Clickable>,
+        );
+
+        // Act
+        userEvent.tab();
+
+        const button = screen.getByTestId("button");
+
+        // Assert
+        expect(button).toHaveFocus();
+    });
+
+    test("allow keyboard navigation when disabled is set", () => {
```

Comment:
Is this test a duplicate of the one before it?

Reply:
ouch good finding, I think that for some reason these tests got duplicated. I'll remove the duplicated ones.